src/guides/v2.3/architecture/archi_perspectives/ABasics_intro.md

```diff
@@ -8,12 +8,12 @@ Magento incorporates the core architectural principles of object-oriented, PHP-b
 
 The following discussion focuses on how these topics apply directly to Magento:
 
-*  Magento technology stack
-*  Magento View Model
-*  Extensibility
-*  Modularity
+*  [Magento technology stack]({{page.baseurl}}/architecture/tech-stack.html)
+*  [Magento View Model]({{page.baseurl}}/extension-dev-guide/view-models.html)
+*  [Extensibility]({{page.baseurl}}/architecture/extensibility.html)
+*  [Modularity]({{page.baseurl}}/architecture/extensibility.html)
```

Comment:
We may point modularity link to the exact section, so

https://devdocs.magento.com/guides/v2.3/architecture/extensibility.html#modularity